### PR TITLE
Prl4101 gate keeping state

### PR DESCRIPTION
**Pull request by harshniavadhannla**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: test/end-to-end/pages/WithdrawApplication.js
Comment (balaganesanhmcts) [inline test/end-to-end/pages/WithdrawApplication.js:1]: Is this withdraw changes required for this Gatekeeping state. Please verify.

File: test/end-to-end/tests/SubmitAndPayCA.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Feature('CA Submit and Pay');
 
-Scenario('Create a Case and Submit/Pay @master', async I => {
+Scenario('Create a Case and Submit/Pay', async I => {
```
Comment (balaganesanhmcts) [inline test/end-to-end/tests/SubmitAndPayCA.js:3]: I think we need @master tag for this test.

File: test/functional/codecept.conf.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 exports.config = {
-  tests: './*_test.js',
+  tests: './GateKeepingState_test.js',
```
Comment (balaganesanhmcts) [inline test/functional/codecept.conf.js:2]: Needs to provide *.js to run based on master or nightly

File: test/smoke/tests/smoke_test.js
Comment (balaganesanhmcts) [inline test/smoke/tests/smoke_test.js:1]: Is those commented line required

File: test/end-to-end/pages/CreateCase.js [hunk 3]
```diff
@@ -203,5 +209,25 @@ module.exports = {
...
     await I.waitForClickable(this.fields.submit);
     await I.retry(retryCount).click(this.fields.submit);
+  },
+
+  async saveTheCaseIdAndSignout() {
+    // I.wait('20');
+    const caseId = normalizeCaseId(await I.grabTextFrom('.alert-message'));
+    console.log(caseId);
+    await I.retry(retryCount).click(this.fields.signOut);
+    I.wait('10');
```
Comment (balaganesanhmcts) [inline test/end-to-end/pages/CreateCase.js:219]: needs to include await

File: test/end-to-end/pages/CreateCase.js [hunk 4]
```diff
@@ -194,7 +200,7 @@ module.exports = {
   },
 
   async fillFormAndSubmit_TSSolicitorApplication() {
-    I.wait('30');
+    I.wait('15');
```
Comment (balaganesanhmcts) [inline test/end-to-end/pages/CreateCase.js:203]: needs to include await

**Review by balaganesanhmcts — APPROVED** [review]

**Review by JamesCutts00 — APPROVED** [review]

**Pull request merged**